src/views/product-tutorials-view/server.ts

```diff
@@ -33,10 +37,10 @@ type ProductTutorialsLayout = Pick<
 
 export interface ProductPageData {
   pageData: {
-    blocks: any // @TODO type
-    showProductSitemap: boolean
-    allCollections: ClientCollection[]
+    blocks: ProductViewBlock[]
+    showProductSitemap?: boolean
   }
+  allCollections: ClientCollection[]
```

Comment:
As mentioned in earlier comment, no tangible change to `allCollections`; moved it alongside `pageData` for clarity in authorable vs "augmenting" data.